src/core/tests/pass/serialization/deterministicity.cpp

```diff
@@ -8,31 +8,35 @@
 
 #include "common_test_utils/common_utils.hpp"
 #include "common_test_utils/file_utils.hpp"
+#include "openvino/opsets/opset1.hpp"
 #include "openvino/pass/serialize.hpp"
 #include "openvino/util/file_util.hpp"
 #include "read_ir.hpp"
 #include "util/test_common.hpp"
 
-class SerializationDeterministicityTest : public ov::test::TestsCommon {
+class DeterministicityCommon {
```

Comment:
Why did you do this change? Why doesn't previous approach work?

Reply:
I want to create new parameterized tests and I need some methods from SerializationDeterministicityTest. So I moved them to DeterministicityCommon class and inherit SerializationDeterministicityTest and new SerializationDeterministicityInputOutputTest from it.